### DISCUSSION TITLE
How to install Kube-bench commands in Windows or Ubuntu machine

### DISCUSSION THREAD

**Discussion by nidiculageorge** [aquasecurity/kube-bench#896] [Questions and Help] [answered]
nidicula@TOPS-ONCALL:~$ kube-bench master
kube-bench: command not found

Could someone help

**Comment by mozillazg** [comment, marked as answer]
@nidiculageorge You can download it from release page: https://github.com/aquasecurity/kube-bench/releases 

BTW, kube-bench don't support Windows yet.

**Comment by nidiculageorge** [comment]
How can we run againsta cluster, for CIS benchmark checking

**Comment by mozillazg** [comment]
@nidiculageorge Just run ./kube-bench or use Docker:

https://github.com/aquasecurity/kube-bench#installing-from-sources
https://github.com/aquasecurity/kube-bench#installing-from-a-container

**Comment by nidiculageorge** [comment]
Hi @mozillazg 

When i try to run the following i could see the following error

kube-bench master

failed to get a set of executables needed for tests: unable to detect running programs for component "kubelet"

**Reply by yoavrotems** [reply]
Hey @nidiculageorge we had a lot of changes one of them is to try always running even `unable to detect running programs for component "kubelet"` We also update our docs and create a documentation site for kube-bench https://aquasecurity.github.io/kube-bench/dev/
